tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+envlist = py38
+isolated_build = true
+
+[testenv]
+commands = python -m pytest
```

Comment:
The flag to report the coverage (`--cov`) when testing is not added since due to the absence of tests `tox.ini` finishes with a failure when trying to discover and run tests.